### ISSUE TITLE
SDK Update Recommended

### ISSUE THREAD

**Issue by vvarda** [jeduan/cordova-plugin-facebook4#770]
Guys, I've received Recommendation message from Facebook for Developers panel.

```
Your app <APP_NAME> uses Facebook SDK for Android v4.25.0 and Facebook SDK for iOS v4.38.1. Please upgrade now to access new privacy features:

Delay Automatic Events
Pause sending events like App Install and App Launch until your app users consent. 

Delay SDK Initialization
Block all network requests until your app users consent.

Upgrade Instructions

For Android: Please update the dynamic version of Facebook SDK in your build.gradle file: implementation 'com.facebook.android:facebook-android-sdk:[5,6)' 

For iOS: Please upgrade to SDK v5.0 and above to access new privacy features.
```
Own config:
```
<plugin name="cordova-plugin-facebook4" spec="4.0.0">
    <variable name="APP_ID" value="<APP_ID>" />
    <variable name="APP_NAME" value="<APP_NAME>" />
    <variable name="FACEBOOK_ANDROID_SDK_VERSION" value="4.25.0" />
</plugin>
```

What action should we take in that case?

**Comment by consciousimages** [comment]
I'm receiving a similar notice, "Your Facebook SDK is out-of-date. Please upgrade to v5.0 for access to new privacy features."  

Looks like 5.0 was released a month ago however, April 30, so still very new and could be buggy.

**Comment by minh1996bk** [comment]
same problem to me.

**Comment by guylando** [comment]
I made a fork/PR which updates SDK to 5.0.2 and fixes compatability https://github.com/jeduan/cordova-plugin-facebook4/pull/751#issuecomment-501596158

**Comment by peterpeterparker** [comment]
The awesome PR of @guylando has been merged and I release v5.0.0 of the plugin which now supports both iOS and Android SDK v5.0.2
